### DISCUSSION TITLE
【深度理财 27】交易期权容易犯的错 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#45] [Announcements]
# 【深度理财 27】交易期权容易犯的错 - forecho's Blog

引言
上个月写了两篇期权干货文章了：

【深度理财 23】什么是期权
【深度理财 24】期权交易基本策略

之前内容比较干，今天来点湿货，我来谈谈这三个月来我交易期权学到的经验。

加一个风险提示：本篇文章仅代表我的个人观点，不构成与其他人相关的任何投资建议。在作出任何投资决定前，投资者应根据自身情况考虑投资产品相关的风险因素，并于需要时咨询专业投资顾问意见。

https://blog.forecho.com/financedeep-27.html

**Comment by hulufei** [comment]
>Sell Put 没到达行权价格会怎么样？
>Sell Put 如果截止日的时候到了行权价格，那么买方就不会行权，你就赚到了权利金。
>Sell Put 如果截止日的时候没到行权价格，那么买方就会行权，你会使用被冻结的资金以行权价买入指定数量的股票，权利金全部归你。

这部分描述是不是有误？

比如我有卖出一张行权价50的Put，到截止日标的价格为60，没到行权价，这时候买方不应该行权吧，行权不是亏了？50卖给我，我转手60就可以卖出。

**Reply by forecho** [reply]
多谢提醒，我换了一种描述方式，之前描述方式不够严谨

**Comment by Dup4** [comment]
> ### Buy Call 没到达行权价格会怎么样？
> * Buy Put 如果截止日的时候到了行权价格，请确保有足够的资金行权，如果资金不够，期权会被强制卖出（平仓）。
> * Buy Put 如果截止日的时候没到行权价格，期权归零。

这段的描述是不是有点 typo，标题说的是 Buy Call，但是内容里面说的是 Buy Put

**Reply by forecho** [reply]
多谢提醒
